tools/torchci/queue_alert.py

```diff
@@ -89,20 +88,11 @@ def filter_long_queues(rockset_result: List[Dict[str, Any]]) -> List[QueueInfo]:
 
 
 def queuing_alert(dry_run: bool) -> None:
-    rs_client = rockset.RocksetClient(
-        host="api.usw2a1.rockset.com", api_key=os.environ["ROCKSET_API_KEY"]
-    )
-    with open(PROD_VERSIONS_FILE) as f:
-        prod_versions = json.load(f)
-
-    # same lambda as the same used by the chart on the hud metrics page
-    response = rs_client.QueryLambdas.execute_query_lambda(
-        query_lambda="queued_jobs_by_label",
-        version=prod_versions["metrics"]["queued_jobs_by_label"],
-        workspace="metrics",
-    )
+    url = "https://hud.pytorch.org/api/clickhouse/queued_jobs_by_label?parameters=%7B%7D%0A"
```

Comment:
Just curious, do you need `%0A` at the end?  The usual logic would be just to have `%7B%7D`

Reply:
Ah good catch, I think its the encoding for new line.  Removed